src/pages/index.tsx

```diff
@@ -139,35 +175,7 @@ const IndexPage = ({ location }: IndexPageProps) => {
           </TeaserGrid>
         </ContentBlockContainer>
 
-        <ContentBlockContainer>
-          <HomePageHeaderBlock
-            topline={t('main.career.topline')}
-            headline={t('main.career.title')}
-            large={true}
-          >
-            {t('main.career.text')}
-          </HomePageHeaderBlock>
-          <TeaserGrid>
-            <Teaser
-              title={t('main.career.teasers.first.title')}
-              linkTo="/career"
-            >
-              {t('main.career.teasers.first.text')}
-            </Teaser>
-            <Teaser
-              title={t('main.career.teasers.second.title')}
-              linkTo="/career"
-            >
-              {t('main.career.teasers.second.text')}
-            </Teaser>
-            <Teaser
-              title={t('main.career.teasers.third.title')}
-              linkTo="/career"
-            >
-              {t('main.career.teasers.third.text')}
-            </Teaser>
-          </TeaserGrid>
-        </ContentBlockContainer>
+        <CareerSection positions={jobPositions} />
```

Comment:
I think it's always worth to divide and conquer even if you only create local components. Do you agree? Make it much more easy to read and navigate.